sonar-python-plugin/src/main/java/org/sonar/plugins/python/pylint/PylintSensor.java

```diff
@@ -19,71 +19,67 @@
  */
 package org.sonar.plugins.python.pylint;
 
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
 import org.apache.commons.io.FileUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.slf4j.LoggerFactory'. ([squid:UselessImportCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.python%3Apython&branch=SONARPY-235b&issues=AWD6XyWqEFfjxkQcPqvw&open=AWD6XyWqEFfjxkQcPqvw)

[ISSUE_KEY]: AWD6XyWqEFfjxkQcPqvw